src/util/cookie.ts

```diff
@@ -27,6 +29,12 @@ export interface CookieAttributes {
      */
     httpOnly?: boolean;
 
+    /**
+     * The SameSite attribute allows you to declare if your cookie should be restricted to a first-party or same-site context.
+     * Refer to {@link https://httpwg.org/http-extensions/draft-ietf-httpbis-rfc6265bis.html#name-samesite-cookiesRFC 6265 section 4.1.2.7.} for more details.
```

Comment:
Just a small thing, there should be a space between the link and 'RFC 6265', also section number is wrong

Reply:
Will fix, thanks